### DISCUSSION TITLE
EXECUTABLE_BLOCK_START - OPTION DISABLE/ENABLE

### DISCUSSION THREAD

**Discussion by RatmanL** [OrcaSlicer/OrcaSlicer#2529] [Dev]
Hello, can I please deactivate the EXECUTABLE_BLOCK_START entry in the G-Code in the settings ? 
With my printer this is a problem to use. 
I have to manually edit the G-code each time and delete the whole EXECUTABLE_BLOCK_START section manually.
PrusaSlicer has an option to deactivate it.
The zeros are entered deliberately. 
I tried deactivating it like this. It didn't work.

Thank you very much.

![image](https://github.com/SoftFever/OrcaSlicer/assets/145371293/e6a06dba-22f1-4cd1-aaea-f27eeb79d7d5)
![image](https://github.com/SoftFever/OrcaSlicer/assets/145371293/0ba5484e-f9d2-4b72-a669-a6e2d5c34958)

**Comment by kocickamia** [comment]
Yep, in Prusaslicer exist option for it. In Orcaslicer doesn't exist any option for it...
![obrazek](https://github.com/SoftFever/OrcaSlicer/assets/4704333/7bb44879-7746-4aec-9968-69942edbd2ec)
Options are: Emit to G-code, Use for time estimate and Ignore.
